app/ads/create/page.tsx

```diff
@@ -343,74 +331,65 @@ export default function CreateAdPage() {
+          alertConfig = {
             title: "You already have an ad with this rate.",
-            message:
+            description:
               "You have another active ad with the same rate for this currency pair and order type. Set a different rate.",
-            type: "warning",
-            actionButtonText: "Update ad",
+            confirmText: "Update ad",
           }
         } else if (error.name === "AdvertOrderRangeOverlap") {
-          errorInfo = {
+          alertConfig = {
             title: "You already have an ad with this range.",
-            message:
+            description:
               "Change the minimum and/or maximum order limit for this ad. The range between these limits must not overlap with another active ad you created for this currency pair and order type.",
-            type: "warning",
-            actionButtonText: "Update ad",
+            confirmText: "Update ad",
           }
         } else if (error.name === "AdvertLimitReached" || error.message === "ad_limit_reached") {
-          errorInfo = {
+          alertConfig = {
             title: "Ad limit reached",
-            message:
+            description:
               "You can have only 3 active ads for this currency pair and order type. Delete one to create a new ad.",
-            type: "error",
-            actionButtonText: "Update ad",
+            confirmText: "Ok",
           }
         } else if (error.name === "InsufficientBalance") {
-          errorInfo = {
+          alertConfig = {
             title: "Insufficient balance",
-            message: "You don't have enough balance to create this ad.",
-            type: "error",
-            actionButtonText: "Update ad",
+            description: "You don't have enough balance to create this ad.",
+            confirmText: "Update ad",
           }
         } else if (error.name === "InvalidExchangeRate" || error.name === "InvalidOrderAmount") {
-          errorInfo = {
+          alertConfig = {
             title: "Invalid values",
-            message: error.message || "Please check your input values.",
-            type: "error",
-            actionButtonText: "Update ad",
+            description: error.message || "Please check your input values.",
+            confirmText: "Update ad",
           }
         } else if (error.name === "AdvertTotalAmountExceeded") {
-          errorInfo = {
+          alertConfig = {
             title: "Amount exceeds balance",
-            message: "The total amount exceeds your available balance. Please enter a smaller amount.",
-            type: "error",
-            actionButtonText: "Update ad",
+            description: "The total amount exceeds your available balance. Please enter a smaller amount.",
+            confirmText: "Update ad",
           }
         } else {
-          errorInfo.message = error.message || errorInfo.message
+          alertConfig.description = error.message || alertConfig.description
         }
       }
```

Comment:
maybe we can move these into its own error file handler. This whole file is quite lengthy already, what would be good is to place it into an error utils file and handle all the errors in there, separate them by functions?